src/index.ts

```diff
@@ -198,7 +200,10 @@ export function withSpid({
         redisClient,
         authorizeRequestTamperer,
         metadataTamperer,
-        getPreValidateResponse(serviceProviderConfig.strictResponseValidation),
+        getPreValidateResponse(
+          serviceProviderConfig.strictResponseValidation,
+          appConfig.applicationInsights
```

Comment:
can we take the AI client from the backend? (so we will share the configuration)

Reply:
Yes, the `appConfig` comes from the backend (provided inside the `withSpid` method)

Reply:
I've thought about this a little bit. Since this may be a general purpose library used by other projects as well (there're no parts tied to the IO platform), it's better to do not introduce any deps to AI (which is Azure specific). That would be better to pass some track() method from the backend that takes something like 
({eventName, type: "error"|"event", data: any })

Reply:
Fixed in a729df3